src/Model/Behavior/RegisterBehavior.php

```diff
@@ -197,8 +197,7 @@ public function getRegisterValidators($options)
         $validateEmail = $options['validate_email'] ?? null;
         $useTos = $options['use_tos'] ?? null;
 
-        $validator = $this->_table->validationDefault(new Validator());
-        $validator = $this->_table->validationRegister($validator);
```

Comment:
This caused problems because the default validation rules would be loaded twice.